pkg/kubelet/container/runtime.go

```diff
@@ -392,6 +392,9 @@ type RunContainerOptions struct {
 	ReadOnly bool
 	// hostname for pod containers
 	Hostname string
+	// EnableHostUserNamespace is true if it is recommended to use the host user namespace in the container
```

Comment:
Comment should be: "When users request other host namespaces, automatically set userns=host for Docker containers" or similar.